index.html

```diff
@@ -1472,6 +1489,48 @@ <h2>
         feature/default allowlist=] is [=default allowlist/'self'=].
       </p>
     </section>
+    <section data-cite="html">
+      <h2>
+        Emulation
+      </h2>
+      <p>
+        For the purposes of user-agent automation and application testing, this
+        document defines geolocation emulations.
+      </p>
+      <p>
+        Each [=top-level traversable=] has an associated
+        <code>"emulated position data"</code>, which is data representing
+        {{GeolocationCoordinates}} or null, initially null.
+      </p>
+      <p>
+        To <dfn class="export">set emulated position data</dfn>, given
+        [=navigable=] |navigable| and an optional |emulated position data|:
+      </p>
+      <ol class="algorithm">
+        <li>Let |traversable| be |navigable|’s [=navigable/top-level traversable=].
```

Comment:
I think at least for BiDi we already assert if a navigable is a top-level traversable, so we probably don't have to do anything here for it?